docker-compose.test.yaml

```diff
@@ -5,7 +5,7 @@ version: '3.7'
 
 services:
   ffc-ce-payment-orchestrator:
-    command: npm run test
```

Comment:
looks like a local modification has sneaked in. There is a docker-compose.test.watch.yaml file that has this configuration if need be.

Reply:
Beat me to it, this has broken the build I think.